packages/website/src/components/Playground.tsx

```diff
@@ -96,73 +112,92 @@ function Playground(): JSX.Element {
         config={state.tsconfig}
         onClose={updateModal}
       />
-      <div className={clsx(styles.options, 'thin-scrollbar')}>
-        <OptionsSelector
-          isLoading={isLoading}
-          state={state}
-          tsVersions={tsVersions}
-          setState={setState}
-        />
-      </div>
       <div className={styles.codeBlocks}>
-        <div className={clsx(styles.sourceCode)}>
-          {isLoading && <Loader />}
-          <EditorTabs
-            tabs={['code', 'tsconfig', 'eslintrc']}
-            activeTab={activeTab}
-            change={setTab}
-            showModal={(): void => setShowModal(activeTab)}
-          />
-          <div className={styles.tabCode}>
-            <EditorEmbed />
+        <SplitPane
+          split="vertical"
+          minSize="10%"
+          defaultSize="20rem"
+          maxSize={
+            20 * parseFloat(getComputedStyle(document.documentElement).fontSize)
+          }
+        >
```

Comment:
Bug report: the options should be responsive when their pane is shrunk. There's a lot of horizontal space that can collapse as needed.

![Screenshot of the playground options being shrunk without collapsing whitespace, so options overflow horizontally](https://user-images.githubusercontent.com/3335181/191344576-66c19b14-0466-4be8-a650-6c421a2a202c.png)

I tried removing the `width: 20rem` from `.options` and that seemed to not break anything!

Reply:
Nice catch. That used to be what set the width of the options panel, but now it's set on the `SplitPane` instance so we don't need it anymore. Removed that line.